pino.js

```diff
@@ -220,6 +220,10 @@ Object.defineProperty(Pino.prototype, 'level', {
   set: Pino.prototype._setLevel
 })
 
+Object.defineProperty(Pino.prototype, 'lscache', {
+  value: copy({}, lscache)
```

Comment:
Ok. Except if you keep a reference to the old instance. So your fix would work. 👍